src/ontology/mondo-ingest-odk.yaml

```diff
@@ -33,6 +33,8 @@ components:
       source: https://data.bioontology.org/ontologies/ICD10CM/submissions/23/download?apikey=8b5b7825-538d-40e0-9e9e-5ab9274a9aeb
     - filename: icd10who.owl
       source: https://github.com/monarch-initiative/icd10who/releases/latest/download/icd10who.ttl
+    - filename: icd11.owl
+      source: https://icd11files.blob.core.windows.net/tmp/whofic-2023-04-08.owl.gz
```

Comment:
## Newer file?
<details><summary>Details</summary>
<p>

### 1. Is there a newer file than this?

### 2. Stable URL?
Do / will they have a stable dateless latest release URL we can use?

### Minor: Automation of finding newer file
If (2) is no then, not that we should, but we could have a GitHub action that checks daily and checks the `YYYY-MM-DD` for the day and, if it finds a new file, updates the config w/ that URL and opens a PR.

</p>
</details>

Reply:
Sounds good. I'll set myself a reminder to make and assign that issue to you later.